src/components/header/share-modal/renderer.tsx

```diff
@@ -39,9 +39,18 @@ class ShareModal extends React.PureComponent<Props, State> {
   }
 
   public onCopy() {
-    this.setState({
-      copied: true,
-    });
+    if (!this.state.copied) {
+      this.setState(
+        {
+          copied: true,
+        },
+        () => {
```

Comment:
this is called when copied is false

Reply:
Yes , this is right , the callback is run after setting the state so this is the logic
1. Set copied to true.
2. Run a timer to make it false.(we know it's true because we have set the state).
3. If copied is false , then only make it true.